src/Constants.js

```diff
@@ -1,5 +1,18 @@
-export const green = '#2BB789'
-export const darkGreen = '#006A42'
+export const green = '#e1e8e9'
+export const darkGreen = '#04555c'
 export const white = '#FFFFFF'
```

Comment:
Add all the colors inside the COLORS const and then change where the code is using the colors.